connect/runtime/src/test/java/org/apache/kafka/connect/runtime/AbstractHerderTest.java

```diff
@@ -135,90 +133,75 @@
     private final int generation = 5;
     private final String connector = "connector";
     private final ConnectorClientConfigOverridePolicy noneConnectorClientConfigOverridePolicy = new NoneConnectorClientConfigOverridePolicy();
+    private Connector insConnector;
 
-    @MockStrict private Worker worker;
-    @MockStrict private WorkerConfigTransformer transformer;
-    @MockStrict private Plugins plugins;
-    @MockStrict private ClassLoader classLoader;
-    @MockStrict private ConfigBackingStore configStore;
-    @MockStrict private StatusBackingStore statusStore;
+    final private Worker worker = mock(Worker.class);
+    final private WorkerConfigTransformer transformer = mock(WorkerConfigTransformer.class);
+    final private Plugins plugins = mock(Plugins.class);
+    final private ClassLoader classLoader = mock(ClassLoader.class);
+    final private ConfigBackingStore configStore = mock(ConfigBackingStore.class);
+    final private StatusBackingStore statusStore = mock(StatusBackingStore.class);
```

Comment:
Nit: we usually write `private final`.

Reply:
Thanks for reminding,it has been modified and test also remove in build.gradle.